README.md

```diff
@@ -1 +1,42 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+# hello-microverse![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
**[Optional]**

- [ ] Kindly remove **line 4** and **line 2** because you already specified your `project name` on **line 1** :blush: to make your readme file professional :+1: